docs/content/getting-started/overview.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Overview | Dagster Docs"
+description: ""
+---
+
+# Welcome to Dagster!
+
+If you’re here, you probably have questions about data, orchestration, Dagster, and how everything fits together.
+
+Let’s start with the basics: **What’s an orchestrator?**
+
+Orchestrators allow you to automatically execute a series of steps and track the results. They not only enable you and your team to see how long steps take to run or how they connect to other steps, but to understand when, where, and why things break.
+
+---
+
+## Why do I need a data orchestrator?
+
+What if you’re working with data? Unlike generic orchestrators that may need to accommodate a variety of use cases, data orchestrators focus on building and managing data pipelines.
+
+Data orchestrators allow you to automatically run, in the correct order, the steps in data pipelines that move, process, and transform data. A data orchestrator helps to ensure that your data assets - such as database tables, reports, language learning models, and so on - are accurate and up-to-date.
+
+That being said, you might be asking yourself...
+
+---
+
+## Why Dagster?
+
+Dagster models pipelines in terms of the data assets they produce and consume, which, by default, brings order and observability to your data platform.
+
+With Dagster, you declare functions in Python that you want to run and the data assets that those functions produce or update. For example, the following image is an example of a graph with several assets:
+
+<!-- ![Example of an asset graph](/images/getting-started/dagster-u-lesson-9-graph.png) -->
+
+<Image
+alt="Example of an asset graph"
+src="/images/getting-started/dagster-u-lesson-9-graph.png"
+width={2474}
+height={1902}
+/>
+
+Dagster’s asset-centric approach to building data pipelines makes it easy to:
+
+- **Model real-world, complex data platforms**. Assets in Dagster can model data produced by any system, such as dbt models, Snowflake tables, or even CSV files.
```

Comment:
Migrating over last comment from Notion:
---
This list is meant to be a set of reasons that Dagster’s asset-centric approach is better than the non-asset-centric approaches taken by general purpose orchestrators, like Airflow and Prefect, right?

General-purpose orchestrators can also handle data produced by any system, so I don’t think this bullet works as a reason for why Dagster’s approach is better.

I do think it's good content for explaining how Dagster's approach works, which we're doing a little bit already in the paragraph above the image.  Thoughts on moving it up there?

Reply:
I missed the part of your comment about moving it up to the previous section. Just did that!